test/source/tests/decrypt.ts

```diff
@@ -760,7 +760,7 @@ XZ8r4OC6sguP/yozWlkG+7dDxsgKQVBENeG6Lw==
       await Util.sleep(5);
       const flowcryptTab = (await browser.browser.pages()).find(p => p.url() === 'https://flowcrypt.com/');
       await flowcryptTab!.waitForSelector("body");
-      flowcryptTab!.on('console', msg => expect((msg as any)._text).to.equal('Opener: null'));
+      flowcryptTab!.on('console', msg => expect((msg as any)._text).to.equal(undefined));
```

Comment:
I don't quite understand what this means and why we need to intercept `console`...
I guess we can simply return `window.opener` in JSON in `evaluate` function on line 765?
like
```
expect(await flowcryptTab!.evaluate(() => `Opener: ${JSON.stringify(window.opener)`)).to.equal('Opener: null');
```

Reply:
Well. Tbh I also was not quite sure about this.
Just fixed test though.
Your suggestion makes sense